asv_bench/benchmarks/utils.py

```diff
@@ -572,3 +573,46 @@ def prepare_io_data(test_filename: str, data_type: str, shapes: list):
         df.to_csv(test_filenames[shape_id], index=False)
 
     return test_filenames
+
+
+CONFIG_FROM_FILE = None
+
+
+def get_benchmark_shapes(bench_id: str, default: list):
```

Comment:
Passing a parameter (`default` in this case) to the function and just returning looks strange a bit. Can we use a different approach?

Reply:
Yes, I'll try another.

Reply:
@YarShev ready for review